crates/libcgroups/src/v1_stub/manager.rs

```diff
@@ -0,0 +1,43 @@
+use crate::common::{AnyCgroupManager, CgroupManager};
```

Comment:
Is there any reason to separate the `v1` and `v1_stub` dir? I'd like to put them together into the `v1` directory like that:

```
v1
|-- manager.rs
L-- stub_manger.rs(or stub.rs)
```

Reply:
Or are all stub files put together into one dir?

Reply:
> Is there any reason to separate the v1 and v1_stub dir?

Because I need these to be different module, so I can use feature switch to control which module is listed. This minimizes the amount of `#[cfg(not(feature = "..."))]` we do. The attribute only applies to a single item in the next line, or I will have to group them into a new mod.

> I'd like to put them together into the v1 directory like that

As mentioned above, we want to put stub vs. not stub into different modules. Likely we will want to maintain the same `pub mod` across the board to keep things easier to find. For example, the `manager.rs` is exposed as `pub mod manager` in both stubs, so it is easy to locate where the files are.

> Or are all stub files put together into one dir?

This can work. I will make the changes.

Reply:
Implemented in 894d424ac43cbf0c860161a83114577197b0b30d

Reply:
👍